interface/main/tabs/templates/tabs_template.php

```diff
@@ -26,7 +26,7 @@
         </div>
         <!-- ko  foreach: tabsList -->
             <div class="tabSpan bgcolor2" data-bind="click: tabClicked, css: {tabNotchosen: !visible()}">
-                <span class="tabTitle" data-bind="text: title, click: tabClicked, css: {tabHidden: !visible()}"></span>
+                <b><span class="tabTitle" data-bind="text: title, click: tabClicked, css: {tabHidden: !visible()}"></span></b>
```

Comment:
I think this is best to do in styling; and rec doing that in a day or 2 after the new styling engine by @d3sandoval is in place:
https://github.com/openemr/openemr/pull/1651

Reply:
this can also be done using a `font-weight` in the `tabTitle` rule

Reply:
OH MAN
I just saw I mistakenly uploaded the sql conf file.
I'll fix this